src/app/test_executive/block_production_timed_accounts_test.ml

```diff
@@ -56,7 +56,7 @@ module Make (Engine : Engine_intf) = struct
   let run network log_engine =
     let open Network in
```

Comment:
are you gonna remove this line? the point of the other changes in this file is so you can remove the `let open Network` line right?